commons/src/main/java/org/apache/causeway/commons/collections/Can.java

```diff
@@ -59,8 +59,10 @@
  * <p>
  * Same idiomatic convention applies: References to {@link Can}
  * should never be initialized to {@code null}.
+ *
  * <p>
- * A {@link Can} must not contain elements equal to {@code null}.
+ * <b>IMPORTANT:</b>A {@link Can} must not contain elements equal to {@code null}.  If you want to store a null within
```

Comment:
I guess the second sentence does not really add any information.

Reply:
That said, `Can<T>` simply does not store `null`. Working around that fact is bad practice. Instead use a different container.

Reply:
updated in subsequent commit direct to `master`.